util/stringutil/string_util_test.go

```diff
@@ -124,3 +124,22 @@ func (s *testStringUtilSuite) TestPatternMatch(c *C) {
 		c.Assert(match, Equals, v.match, Commentf("%v", v))
 	}
 }
+
+func (s *testStringUtilSuite) TestIsExactMatch(c *C) {
+	defer testleak.AfterTest(c)()
+	tbl := []struct {
+		patTypes   []byte
+		exactMatch bool
+	}{
+		{nil, true},
+		{[]byte{}, true},
+		{[]byte{1}, true},
+		{[]byte{1, 1}, true},
+		{[]byte{1, 2}, false},
+		{[]byte{1, 3}, false},
+		{[]byte{1, 2, 3}, false},
+	}
```

Comment:
Better to use original string as input and `CompilePattern` to convert the string, then use the result to test `IsExactMatch`.

Reply:
ok, Done.